subsys/testsuite/include/ztest_unittest.ld

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+SECTIONS
+{
+	.data.ztest_suite_node_area : ALIGN(4)
```

Comment:
Does this need to use _ZTEST_ALIGN?

Reply:
The `ztest_unittest.ld` is used in a completely different build which only runs native posix. So there's no need for it to be anything other than 4.